src/Question.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+class Question extends Component {
+  render() {
+    return (
+      <li className="list">
+        <Router>
+          <Link to={`/question/${this.props.data.id}`}>
+            {this.props.data.question}
+          </Link>
+        </Router>
+      </li>
+    );
+  }
+}
+
+Question.propTypes = {
+  data: PropTypes.shape({
```

Comment:
Why is this prop called "data" and not "value", for example?
"value" is also universal enough.